libraries/rush-lib/webpack.config.js

```diff
@@ -86,4 +82,16 @@ module.exports = () => {
       }
     ]
   };
+
+  DeepImportsCompatPlugin.applyToWebpackConfiguration(configuration, {
```

Comment:
I am very much not a fan of this API. It breaks the normal contract for how one instantiates webpack plugins and adds them to a webpack compilation.